common/proc.c

```diff
@@ -91,34 +95,46 @@ proc_status_new(pid_t pid)
 	IF_FALSE_GOTO(n == 1, error);
 	TRACE("Parsed state for %d: %c", pid, status->state);
 
-	tmp = strstr(buf, "\nPid:");
```

Comment:
change buf in strstr(buf, "\nState:") also to tmp?

Reply:
nop as tmp is used beforehand on another string namely status->name in L87
Thus, tmp is not an internal pointer in the buf at this place.

Reply:
ah now i get it. ack then.